pkg/geo/geopb/geopb.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+syntax = "proto3";
+package cockroach.geopb;
+option go_package = "geopb";
+
+// ShapeType is the type of a spatial object. Each of these corresponds to a
```

Comment:
I feel as though `ShapeType` is a very `S2` inspired name. Maybe `SpatialDataType`?

Reply:
the spec calls it `GeometryType`